nodebuilder/tests/swamp/config.go

```diff
@@ -35,3 +37,20 @@ func WithBlockTime(t time.Duration) Option {
 		c.Tendermint.Consensus.SkipTimeoutCommit = false
 	}
 }
+
+// WithLogLevel overrides basic log levels for test purposes
+func WithLogLevel(components []string, levels []string) Option {
+	return func(_ *Components) {
+		if len(components) != len(levels) && len(levels) != 1 {
+			panic("inconsistent amount of components and levels")
```

Comment:
can we log before we panic where was the mismatch by printing all levels + components? 🙏 

This gives much better assurance for future debugging sessions 😅